packages/sync/src/Queue.php

```diff
@@ -140,10 +204,17 @@ function has_any_items() {
 			)
 		);
 
-		return ( $value === '1' );
+		return ( '1' === $value );
 	}
 
-	function checkout( $buffer_size ) {
+	/**
+	 * Used to checkout the queue.
```

Comment:
Would be useful to further clarify what we return here, as there are plenty of different return options.